clients/python/tests/fuzz_api/model_registry/test_mr_stateless.py

```diff
@@ -233,38 +239,333 @@ def check_null_byte_validation_mr(ctx, response, case):
     ],
 )
 @pytest.mark.fuzz
-def test_mr_api_stateless(auth_headers: dict, case: schemathesis.Case, verify_ssl: bool) -> None:
-    """Test the Model Registry API endpoints.
+def test_mr_api_stateless_get_models(auth_headers: dict, case: schemathesis.Case, verify_ssl: bool) -> None:
+    """Test GET endpoints for RegisteredModels and ModelVersions."""
+    try:
+        case.call_and_validate(headers=auth_headers, verify=verify_ssl)
+    except Exception as e:
+        request_has_null_bytes = False
+        if case.query:
+            request_has_null_bytes = contains_null_bytes(case.query)
+        if not request_has_null_bytes and hasattr(case, "body") and case.body:
+            try:
+                if isinstance(case.body, (str, bytes)):
+                    body_str = case.body.decode("utf-8") if isinstance(case.body, bytes) else case.body
+                    request_has_null_bytes = contains_null_bytes(body_str)
+            except (UnicodeDecodeError, AttributeError):
+                pass
+        if request_has_null_bytes and ("400" in str(e) or "CheckFailed" in str(type(e).__name__)):
+            logging.info(f"Expected 400 error for null bytes: {case.path}")
+            pytest.skip("Expected validation error for null bytes")
+        else:
+            raise
```

Comment:
This can be a utility function and be reused in  test_mr_api_stateless_get_artifacts(), test_mr_api_stateless_get_experiments() etc. What do you think?